app/javascript/notifications/backend.js

```diff
@@ -24,28 +24,17 @@ export function convert(resource) {
 }
 
 export function load(useLimit) {
-  const promises = [];
   const limitFragment = useLimit ? `&limit=${maxNotifications}` : '';
 
-  promises.push(API.get(`/api/notifications?expand=resources&attributes=details&sort_by=id&sort_order=desc${limitFragment}`)
+  return API.get(`/api/notifications?expand=resources&attributes=details&sort_by=id&sort_order=desc${limitFragment}`)
     .then((data) => {
       const notifications = data.resources.map(convert);
 
       return {
         notifications,
-        subcount: data.subcount,
+        subcount: data.count
```

Comment:
I was going to change this key to `count` also, but that feels like a much more invasive change, so I left it for now.